<launch>



  <arg name="modelmain" default="$(find prototyping_description)urdf/output.xacro"/>

  <include file="launch/gazebo.launch">
    
  </include>

  <rosparam command="load"
        file= "config/joints.yaml"
  ns="joint_state_controller" />

  <rosparam command="load"
  file= "config/L_finger.yaml"
  ns="left_controller" />

  <rosparam command="load"
  file= "config/R_finger.yaml"
  ns="right_controller" />

  <rosparam command="load"
  file = "config/arm.yaml"
  ns="arm_controller" />


  <node pkg="prototyping_description" name="joint_control_script" type="joint_control_script.py" />
    
  <node pkg="prototyping_description" name="arm_control_script" type="arm_control_script.py" />

  <node pkg="prototyping_description" name="description_logger" type="description_logger.py"/>

  <node name="base_joint_controller_spawner" pkg="controller_manager" type="spawner"
  args="joint_state_controller
        left_controller
        right_controller
        arm_controller
              --shutdown-timeout 3" />

</launch>
